sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -728,7 +735,7 @@ private void processMinidumpFiles() {
              */
             errorLog.setUserId(UserIdContext.getInstance().getUserId());
             try {
-                errorLog.setDevice(getDeviceInfo(mContext));
+                errorLog.setDevice(DeviceInfoHelper.getDeviceInfoByTimestamp(minidumpDate));
```

Comment:
Let's use it also in the case when java crashes were unable to save this info at crash time